### PR TITLE
Sponsors and card

### PR DESCRIPTION
**Pull request by SamCarter14**

### CODE REVIEW

File: 2023/css/styles.css [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
-    --clr-ctn: #e3753f;
-    --clr-txt2: #e3d46f;
+    --clr-ctn: #78B4BE;
+    --clr-txt2: #3d3d3d;
 }
 
 * {
     margin: 0;
     padding: 0;
+    box-sizing: border-box;
```
Comment (JT2M0L3Y) [inline 2023/css/styles.css:13]: curious what this is for

File: .DS_Store
Comment (JT2M0L3Y) [inline .DS_Store:1]: get this out...

File: 2023/.DS_Store
Comment (JT2M0L3Y) [inline 2023/.DS_Store:1]: get this out as well...

File: 2023/index.html [hunk 2]
```diff
@@ -85,19 +127,61 @@ <h1>Gonzaga Hackathon</h1>
...
-                <label for="content-1" class="carousel-dot" id="content-dot-1"></label>
-                <label for="content-2" class="carousel-dot" id="content-dot-2"></label>
-                <label for="content-3" class="carousel-dot" id="content-dot-3"></label>
-                <label for="content-4" class="carousel-dot" id="content-dot-4"></label>
-            </div>
         </ul>
     </section>
+</div>
 
+<div class="carousel-dots">
```
Comment (JT2M0L3Y) [inline 2023/index.html:130]: this was in the \<ul>, what is the purpose of moving it out?

File: 2023/pictures/.DS_Store
Comment (JT2M0L3Y) [inline 2023/pictures/.DS_Store:1]: get rid of...

File: 2023/public/.DS_Store
Comment (JT2M0L3Y) [inline 2023/public/.DS_Store:1]: get rid of...

**Review by JT2M0L3Y — CHANGES_REQUESTED** [review]
Biggest thing to improve is moving everything in /pictures into /public. The public directory is traditionally used for any files that do not need to be handled by a compiler or processing body.

**Review by Colexeco — APPROVED** [review]
Just lmk when all of Jonathan's stuff is resolved and I deal with merge conflicts

File: 2023/index.html (hunk 2 above)
Comment (Colexeco) [reply 2023/index.html:134]: In the what?

Comment (SamCarter14) [reply 2023/index.html:130]: what is this saying?

File: 2023/css/styles.css (hunk 1 above)
Comment (SamCarter14) [reply 2023/css/styles.css:13]: pretty sure that was to get the sponsors under the cards, not positive... ill look closer into it

File: 2023/index.html (hunk 2 above)
Comment (JT2M0L3Y) [reply 2023/index.html:130]: I am wondering why the dots at the bottom of the carousel were moved outside of the list element. They were previously a part of the list. (referencing the div "carousel-dots")

Comment (SamCarter14) [reply 2023/index.html:130]: They were disappearing/going in the wrong spot since I added another div, and this was my solution.

Comment (JT2M0L3Y) [reply 2023/index.html:130]: ok, thanks

**Review by JT2M0L3Y — APPROVED** [review]
@Colexeco @SamCarter14, I fixed the merge conflicts early, hope that's okay! I left the merge back to main for someone else after confirmation. The issue with the /pictures directory is still present.

**Comment by SamCarter14** [comment]
Does it have the correct google form link? mine had the old link.

**Comment by Colexeco** [comment]
I believe there is no newer form link yet

**Comment by Colexeco** [comment]
I'm gonna merge now. Thanks for fixing merge conflicts Jonathan!

**Review by Colexeco — APPROVED** [review]

**Comment by JT2M0L3Y** [comment]
I will open up new issues for the google form link and the pictures directory.

**Pull request merged**